docs/docs/index.mdx

```diff
@@ -0,0 +1,67 @@
+
+:::info
+
+Noir is backend agnostic, which means it makes no assumptions on which proving backend powers the ZK proof. Being the language that powers [Aztec Contracts](https://docs.aztec.network/dev_docs/contracts/main), it defaults to Aztec's Barretenberg proving backend.
+
+However, the ACIR output can be transformed to be compatible with other PLONK-based backends, or into a [rank-1 constraint system](https://www.rareskills.io/post/rank-1-constraint-system) suitable for backends such as Arkwork's Marlin.
+
+:::
+
+## Who is Noir for?
+
+Noir can be used both in complex cloud-based backends and in user's smartphones, requiring no knowledge on the underlying math or cryptography. From authorization systems that keep a password in the user's device, to complex on-chain verification of recursive proofs, Noir is designed to abstract away complexity without any significant overhead. Here are some examples of situations where Noir can be used:
+
+<Tabs>
+  <TabItem value="Aztec Contracts" label="Aztec Contracts" default>
+    <img src={require("@site/static/img/aztec_logo.png").default} style={{display: "block", width: "50%", margin: "2rem auto"}} alt="Noir Logo"  />
+    
+    Aztec Contracts leverage Noir to allow for the storage and execution of private information. Writing an Aztec Contract is as easy as writing Noir, and Aztec developers can easily interact with the network storage and execution through the [Aztec.nr](https://docs.aztec.network/dev_docs/contracts/main) library.
+  </TabItem>
+  <TabItem value="Solidity Verifiers" label="Solidity Verifiers">
+    <img src={require("@site/static/img/solidity_verifier_ex.png").default} style={{display: "block", margin: "2rem auto"}} alt="Soliditry Verifier Example"  />
+    Noir can auto-generate Solidity verifier contracts that verify Noir proofs. This allows for non-interactive verification of proofs containing private information in an immutable system. This feature powers a multitude of use-case scenarios, from P2P chess tournaments, to [Aztec Layer-2 Blockchain](https://docs.aztec.network/)
+  </TabItem>
+  <TabItem value="Full-Stack Development" label="Full-Stack Development">
+    Aztec Labs developed NoirJS, an easy interface to generate and verify Noir proofs in a Javascript environment. This allows for Noir to be used in webpages, mobile apps, games, and any other environment supporting JS execution in a standalone manner.
+  </TabItem>
+</Tabs>
+
+
+## Libraries
+
+Noir is meant to be easy to extend by simply importing Noir libraries just like in Rust. 
+The [awesome-noir repo](https://github.com/noir-lang/awesome-noir#libraries) is a collection of libraries developed by the Noir community.
+Writing a new library is easy and makes code be composable and easy to reuse. See the section on [dependencies](noir/modules_packages_crates/dependencies.md) for more information.
```

Comment:
Do we want to point readers to various sections at the bottom of this page? We could direct them to Language Concepts, Explainers and Getting Started?